### DISCUSSION TITLE
I want to use multiple database

### DISCUSSION THREAD

**Discussion by raksrivastava** [prisma/prisma#9166] [Q&A]
I am not able to use multiple database in same application . How can we use multiple data sources.
Can we generate multiple "schema.prisma" for different  database connections.

**Comment by chenkaiC4** [comment]
same with me, i want to use the same database with multiple schema, not just one

**Reply by janpio** [reply]
This is not supported right now, but the feature request is tracked at https://github.com/prisma/prisma/issues/1122 @chenkaiC4

**Comment by janpio** [comment]
This is not supported right now @raksrivastava. The feature request is tracked at https://github.com/prisma/prisma/issues/2443.

You _can_ create two completely separate Prisma Client instances in your project though to achieve something similar: https://github.com/prisma/prisma/issues/2443#issuecomment-630679118

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
